lib/content-services/src/lib/i18n/en.json

```diff
@@ -487,7 +488,11 @@
       "MODIFIED_DATE": "Modified Date",
       "CONTENT_TYPE": "Content Type",
       "TAGS": "Tags",
-      "ADD_TAG_TOOLTIP": "Add tag"
+      "ADD_TAG_TOOLTIP": "Add tag",
+      "HEADER_TITLE": "General info",
+      "NO_TAGS_ADDED": "There are currently no tags added",
+      "NO_ITEMS_MESSAGE": "There are currently no {{ groupTitle }} added",
+      "SNACKBAR_MESSAGE": "Save or discard changes to continue"
```

Comment:
the key is too generic and confusing and does not give clue what the message is about